# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/agg/hotfix-automake.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- ./configure.ac.vanilla	2025-09-10 11:11:39.039000000 +0200
+++ ./configure.ac	2025-09-10 11:13:06.681991686 +0200
@@ -157,10 +157,10 @@
    src/platform/win32/Makefile
    src/platform/BeOS/Makefile
    src/platform/AmigaOS/Makefile
-   include/Makefile
-   include/ctrl/Makefile
-   include/util/Makefile
-   include/platform/Makefile
+   include/agg/Makefile
+   include/agg/ctrl/Makefile
+   include/agg/util/Makefile
+   include/agg/platform/Makefile
    examples/Makefile
 ) 
 
--- agg/Makefile.am.vanilla	2025-10-23 11:46:42.011480424 +0200
+++ agg/Makefile.am	2025-10-23 11:46:46.309462337 +0200
@@ -1,4 +1,4 @@
-SUBDIRS = gpc src font_freetype font_win32_tt include examples
+SUBDIRS = gpc src font_freetype font_win32_tt include/agg
 
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = libagg.pc
